### PR TITLE
Disable access to network bpf() syscall from unprivileged processes.

### PR DESCRIPTION
**Pull request by adelton**
#### Description:

- Disable access to network bpf() syscall from unprivileged processes.

#### Rationale:

- Loading and accessing the packet filters programs and maps using the bpf() syscall has the potential of revealing sensitive information about the kernel state.

### CODE REVIEW

File: linux_os/guide/system/permissions/restrictions/sysctl_kernel_unprivileged_bpf_disabled/rule.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Loading and accessing the packet filters programs and maps using the bpf()
+    syscall has the potential of revealing sensitive information about the kernel state.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 82974-7
+
+references:
+    anssi: NT28(R25)
```
Comment (jan-cerny) [inline linux_os/guide/system/permissions/restrictions/sysctl_kernel_unprivileged_bpf_disabled/rule.yml]: ANSSI NT28 R25 says: 

> It is recommended to load the Yama security module at startup (by example passing the security = yama argument to the kernel) and configure the sysctl kernel.yama.ptrace_scope to a value of at least 1.

I think this rule isn't related to YAMA so this reference should be removed. I haven't found any mention of bpf in ANSSI NT28.

**Comment by jan-cerny** [comment]
Fixes #4493

**Comment by adelton** [comment]
Removed that `anssi: NT28(R25)` reference, rebased on master -> 2f9039139360627995ac7bd01f5294a993fde38c.

**Comment by adelton** [comment]
Rebased on master -> 6f4281133b6d9c504b99a2b8c94d9757a94a0144.

**Comment by jan-cerny** [comment]
@adelton thank you

**Pull request merged**